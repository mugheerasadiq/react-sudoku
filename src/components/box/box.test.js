import React from "react";
import renderer from "react-test-renderer";

import Box from "./Box.index";

it("All props are passed", () => {
  const tree = renderer
    .create(
      <Box row={1} col={1} value={1} onChange={() => console.log("test")} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("No value is passed", () => {
  const tree = renderer.create(<Box />).toJSON();
  expect(tree).toMatchSnapshot();
});
