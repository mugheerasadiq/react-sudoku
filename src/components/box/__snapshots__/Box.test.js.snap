// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All props are passed 1`] = `
<td>
  <input
    name="1,1"
    onChange={[Function]}
    style={
      Object {
        "backgroundColor": "#90CAF9",
        "fontSize": "3em",
        "fontWeight": "150",
      }
    }
    value={1}
  />
</td>
`;

exports[`No value is passed 1`] = `
<td>
  <input
    name="undefined,undefined"
    style={
      Object {
        "backgroundColor": undefined,
        "fontSize": "3em",
        "fontWeight": "150",
      }
    }
  />
</td>
`;
